examples/linear_model/plot_lasso_coordinate_descent_path.py

```diff
@@ -20,6 +18,8 @@
 from sklearn.linear_model import lasso_path, enet_path
 from sklearn import datasets
 
+print(__doc__)
+
```

Comment:
`__doc__` is set to `''` by sphinx-gallery when the example is run. https://github.com/sphinx-gallery/sphinx-gallery/blob/2c9d45b0cb261873fd7706f15badd8d03c43c58a/sphinx_gallery/gen_rst.py#L581.

Also it seems that sphinx-gallery does a `.strip()` on the output, so `print(__doc__)` as the first thing that writes to stdout in the script will not show up in the sphinx-gallery output section of the HTML.